packages/mdc-tab-indicator/README.md

```diff
@@ -0,0 +1,128 @@
+<!--docs:
+title: "Tab Indicator"
+layout: detail
+section: components
+excerpt: "Tab Indicator is a visual guide that shows which Tab is active"
+iconId: tab
+path: /catalog/tab/
+-->
+
+# Tab Indicator
+
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components-web.appspot.com/tab-indicator.html">
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/tab-indicator.png" width="363" alt="Tab indicator screenshot">
+  </a>
+</div>-->
+
+Tab Indicator is a visual guide that shows which Tab is active
+
+## Design & API Documentation
+
+<!--
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/tabs.html">Material Design guidelines: Tab Indicator</a>
+  </li>
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components-web.appspot.com/tab-indicator.html">Demo</a>
+  </li>
+</ul>
+-->
+
+## Installation
+```
+npm install --save @material/tab-indicator
+```
+
+## Usage
+
+### HTML Structure
+
+```html
+<span class="mdc-tab-indicator"></span>
+```
+
+#### Sliding Bar Indicator
+```html
+<span class="mdc-tab-indicator">
+  <span class="mdc-tab-indicator__bar"></span>
+</span>
+```
+
+#### Fading Icon Indicator
+
+We recommend you load [Material Icons](https://material.io/icons/) from Google Fonts. However, users are free to use whatever icons they like.
```

Comment:
You can use [Material Icons](https://material.io/icons/) from Google Fonts within your Fading Icon Indicator, or you can use your own icons